README.md

```diff
@@ -16,6 +16,13 @@ cd csvtofhir
 # create virtual environment and create an "editable" install
 python3 -m venv venv && source venv/bin/activate && python3 -m pip install --upgrade pip setuptools
 python3 -m pip install -e .[dev]
+
+# From the venv: install the packages
+python setup.py install
```

Comment:
The `setup.py install` step is redundant as the libraries are install in the virtual environment by the previous command, `python3 -m pip install -e .[dev]`

A note we could add though is that for folks using the Z shell on OS X or Linux you will need to use double-quotes around the command:
```shell
python3 -m pip install -e ".[dev]"
```
The square brackets need to be escaped or quoted, due to the way Z shell expansion works. Here's a handy [Stack Overflow Article](https://stackoverflow.com/questions/30539798/zsh-no-matches-found-requestssecurity)

Reply:
Ahh the `python3 -m pip install -e .[dev]` failed for me.  That's why I had to figure out the individual installs.
Yes, adding the note to use quotes would be helpful.